src/button/Button.tsx

```diff
@@ -76,10 +75,12 @@ function Button({
       {children}
 
       {shouldDisplaySpinner && (
-        <Spinner
-          customClassName={"button__spinner"}
-          aria-label={"Button spinner visible. Button inactivated."}
-        />
+        <div className={"button__spinner__container"}>
```

Comment:
I think the class name should be `button__spinner-container` (meaning `spinner-container` inside `button`)
If we use `__`, it looks like container is the child of spinner,
and '--' is for modifiers (`--is-active` etc.),
so `-` looks more suitable here

Reply:
Yes, you're right. As Ediz said, we can use `button__spinner-container` then.